test/quote_validator_test.ts

```diff
@@ -126,7 +161,7 @@ describe(SUITE_NAME, () => {
                 timeFirstSeen: new Date(new Date().getTime() - ONE_SECOND_MS * 30),
             });
 
-            const orders = [MAKER1_ADDRESS, MAKER2_ADDRESS, MAKER3_ADDRESS].map(makerAddress => {
```

Comment:
What changes did you apply here?

Reply:
looks like linter change: parens around the input